### PR TITLE
Mark PatternAcceptanceTest as ported to the TCK

### PR DESCRIPTION
**Pull request by Mats-SX**
Should only be merged after https://github.com/opencypher/openCypher/pull/66 is merged.

### CODE REVIEW

File: community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/PatternExpressionAcceptanceTest.scala [hunk 1]
```diff
@@ -315,6 +336,7 @@ class PatternExpressionAcceptanceTest extends ExecutionEngineFunSuite with Match
     ))
   }
 
+  // TCK'd
```
Comment (petraselmer) [inline community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/PatternExpressionAcceptanceTest.scala]: I don't know if we should mark these tests - ones which include introspecting the plan, as per line 353 - as TCK'ed...that is, the actual test is TCK'ed, but not every aspect as envisaged by this test. I have come across similar. Maybe we ought to discuss...

Comment (Mats-SX) [inline community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/PatternExpressionAcceptanceTest.scala]: Yes, perhaps these tests should receive a different annotation.

**Comment by Mats-SX** [comment]
@petraselmer I've done this now in a bit of a different way. I removed all result assertions, and moved the tests with plan assertions to a differently named test class. The ported tests are removed, as per our new strategy.

@boggle I'd be happy for you to see this as well.

**Comment by Mats-SX** [comment]
Deal with case expression.

**Comment by Mats-SX** [comment]
@boggle This should be mergable now. Only acceptance tests with openCypher features are removed.

**Pull request merged**